benchmarks/basic.js

```diff
@@ -8,6 +8,9 @@ var winston = require('winston')
 var fs = require('fs')
 var dest = fs.createWriteStream('/dev/null')
 var plog = pino(dest)
+delete require.cache[require.resolve('../')]
```

Comment:
for a fair test (results are different otherwise) - deliberately loading two separate instances of pino (avoids polymorphic blah blah etc.) - I don't think it matters for basic but it matters elsewhere, I did it on all benchmarks to be sure